src/uicontrols.cpp

```diff
@@ -0,0 +1,74 @@
+#include "uicontrols.h"
+
+#include <QString>
+
+namespace {
+const QString kSkinGroup = QStringLiteral("[Skin]");
+}
+
+UIControls::UIControls()
+        : m_effectRack1Show(ConfigKey(QStringLiteral("[EffectRack1]"),
+                                    QStringLiteral("show")),
+                  true,
+                  true),
+          m_libraryShowCoverArt(ConfigKey(QStringLiteral("[Library]"),
+                                        QStringLiteral("show_coverart")),
+                  true,
+                  true),
+          m_microphoneShowMicrophone(ConfigKey(QStringLiteral("[Microphone]"),
+                                             QStringLiteral("show_microphone")),
+                  true,
+                  true),
+          m_previewDeckShowPreviewDeck(
+                  ConfigKey(QStringLiteral("[PreviewDeck]"),
+                          QStringLiteral("show_previewdeck")),
+                  true,
+                  true),
+          m_samplersShowSamplers(ConfigKey(QStringLiteral("[Samplers]"),
+                                         QStringLiteral("show_samplers")),
+                  true,
+                  true),
+          m_skinShow4EffectUnits(
+                  ConfigKey(kSkinGroup, QStringLiteral("show_4effectunits")),
+                  true,
+                  false),
+          m_skinShowCoverArt(
+                  ConfigKey(kSkinGroup, QStringLiteral("show_coverart")),
+                  true,
+                  true),
+          m_skinShowMaximizedLibrary(
+                  ConfigKey(
+                          kSkinGroup, QStringLiteral("show_maximized_library")),
+                  true,
+                  false),
+          m_skinShowMixer(ConfigKey(kSkinGroup, QStringLiteral("show_mixer")),
+                  true,
+                  true),
+          m_skinShowSettings(
+                  ConfigKey(kSkinGroup, QStringLiteral("show_settings")),
+                  false,
+                  false),
+          m_skinShowSpinnies(
+                  ConfigKey(kSkinGroup, QStringLiteral("show_spinnies")),
+                  true,
+                  true),
+          m_vinylControlShowVinylControl(
+                  ConfigKey(QStringLiteral("[VinylControl]"),
+                          QStringLiteral("show_vinylcontrol")),
```

Comment:
controlpickermenu.cpp still uses `VINYL_PREF_KEY`
Shouldn't we move this to the `Skin` group as well?

Reply:
Sounds reasonable. The same applies to the other COs in `uicontrols.cpp`.

I only the COs that had to be renamed, but if we want to make stuff consistent, I would propose this.

| Old Name                           | New Name
| ---------------------------------- | ----------------------------------
| `[EffectRack1],show`               | `[Skin],show_effectrack`
| `[Library],show_coverart`          | `[Skin],show_library_coverart`
| `[Microphone],show_microphone`     | `[Skin],show_microphone`
| `[PreviewDeck],show_previewdeck`   | `[Skin],show_previewdeck`
| `[Samplers],show_samplers`         | `[Skin],show_samplers`
| `[VinylControl],show_vinylcontrol` | `[Skin],show_vinylcontrol`

Reply:
Yeah, sure, go ahead! IMO it doesn't make much sense binding skin controls to these engine groups.
I didn't dare to propose changing all those atm to not slow down your renaming rush ; )

Reply:
> Yeah, sure, go ahead!

If everyone agrees of course!